lib/slack.py

```diff
@@ -32,10 +32,14 @@ def __init__(self):
         # Cache the client in memory
         self._client = None
 
-    def send_message(self, params):
+    def send_message(self, params, is_ephemeral=False):
```

Comment:
Rather than have ephemerality as a boolean, let's accept "Message" or "Ephemeral" as a string argument and append it to the api_call's first arg. That way we need not have an if statement at all (and that way we have one `api_call` to rule them all)!

Another option would be to have a one line if statement in the API call like so:

`client.api_call("chat.postEphemeral" if is_ephemeral else "chat.postMessage", **params)`.

I'll leave it up to your discretion which one you want to go with, (the former saves us the if check entirely).